app/view/twig/_base/_listing.twig

```diff
@@ -301,13 +301,13 @@ row_header:     prop.extended and (prop.first or (prop.has_groupname and prop.ne
                     {% for taxonomyslug, values in content.taxonomy %}
                         {% if values|length > 1 %}
                             <li>
-                                <a class="nolink">{{ config.get('taxonomy')[taxonomyslug].name }}:
+                                <a class="nolink">{{ app.config.get('taxonomy')[taxonomyslug].name }}:
```

Comment:
You had it marked as so, and the template base was split between them … what's it going to be?